apps/web/src/components/PoolFilterInput/PoolFilterInput.tsx

```diff
@@ -3,7 +3,10 @@ import debounce from "lodash/debounce";
 import { useState } from "react";
 
 import { Combobox } from "@gc-digital-talent/forms";
-import { PoolFilterInput, Scalars } from "@gc-digital-talent/graphql";
+import type {
+  PoolFilterInput as TPoolFilterInput,
```

Comment:
Why the `T` prefix.  That's unconventional, no?

Reply:
Unconventional for us, but not typescript in general. 

However, this is only required to avoid name collision. I couldn't think of a better name for the component so I just aliased it with a prefix. I would love to avoid this so I am more than happy to take suggestions! :smiley:

Reply:
Oh yeah, I see it now.  A better name doesn't come to mind so this is as good as any.

Reply:
Yeah, I'm kind of disappointed this did not get caught when building with webpack :confounded: